### PR TITLE
kyle first pull request

### PR DESCRIPTION
**Pull request by imkyleyu**

### CODE REVIEW

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Looks good! Please take a look at #23 and fix your profile accordingly.

File: page-officers.hbs [hunk 1]
```diff
@@ -332,6 +332,15 @@ into the {body} of the default.hbs template --}}
                     <a target="_blank" class="icon-github" href="https://github.com/DarKnight0102/"></a>
                   </div>
                 </div>
+                <div class="ieee-officer-card">
+                  <img src="{{asset 'officers/Website/kyle_yu.png'}}" alt="profilepic"/>
```
Comment (PotatoParser) [inline page-officers.hbs]: The url needs changing here

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Looks good, just a slight correction is needed.

File: page-officers.hbs (hunk 1 above)
Comment (imkyleyu) [reply page-officers.hbs]: Oops... sorry about that.

**Review by PotatoParser — APPROVED** [review]
Great!

**Pull request merged**